examples/samplers/fast_sampling_with_jax_and_numba.ipynb

```diff
@@ -19,18 +19,58 @@
    "cell_type": "markdown",
```

Comment:
<p>Any reason why this model specifically?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/794/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>